styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+}
+
+/* @font-face {
+  font-family: 'Inter';
+  src: url('../fonts/inter-v12-latin-regular.eot');
+} */
```

Comment:
Great job @taldr27 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is not regarded as a good practice.

Reply:
Ok, thank you.